justfile

```diff
@@ -65,21 +65,21 @@ test-rust:
     cargo test --no-fail-fast
 
 # Supported presets: all, rolldown, rollup
-test-node preset="all" *args="":
+test-node preset="all" *args="": _build-native-debug
+    just _test-node-{{preset}} {{args}}
+
+test-node-only preset="all" *args="":
```

Comment:
`only` is confusing. But I can't think of a good name either. You might able rename these `_test-node-all` to `_run-node-test-all` and use `just _run-node-test-all`. I'm open to both.

Reply:
but the command with the prefix `_` will lose auto-completion.

Reply:
Get it. I kind of feel being limited while writing these commands. These `_test-node-all` is kind of hacky. A real cli would be more intuative on applying args. Such as `just test-node --no-build`. However, so far let's just bare  `just`.